src/services/challenge-pull/index.ts

```diff
@@ -385,4 +385,44 @@ export default class ChallengePullService implements IChallengePullService {
       message: ChallengePullMessage.Rewarded,
     };
   }
+
+  public async awardWhenPullClosedAndContainClose(
+    pullPayload: PullPayload
+  ): Promise<boolean> {
+    const issueNumber = findLinkedIssueNumber(pullPayload.pull.body);
+    const closeIndex = pullPayload.pull.body.toLowerCase().indexOf("close");
```

Comment:
We should match Issue Number: close xxx as a whole, not the close keyword directly.

Reply:
I use re to solve this problem, please help me check the code again

Reply:
You can use https://github.com/ti-community-infra/ti-challenge-bot/blob/2377b98b34c0eeeebfae7f1b930d11b2c8fde6c6/src/services/utils/PullUtil.ts#L4.

Reply:
After thinking about it, I think we should build a new method